maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -46,6 +46,9 @@ enum class DragState {
     START, DRAG, END
 }
 
+@Immutable
+class MarkerWrapper(val value: Marker? = null)
```

Comment:
I made a wrapper class to add `Immutable` annotation. Please let me know if you have any good ideas.